lms/templates/help_modal.html

```diff
@@ -18,207 +18,191 @@
   <a href="#help-modal" rel="leanModal" role="button">${_("Support")}</a>
 </div>
 
-<section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
-  <div class="inner-wrapper" id="help_wrapper">
+<section id="help-modal" class="modal" aria-hidden="true" role="dialog" tabindex="-1" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
+  <div class="inner-wrapper">
     ## TODO: find a way to refactor this
-    <button class="close-modal "tabindex="0">
+    <button class="close-modal" tabindex="0">
       <span class="icon fa fa-remove" aria-hidden="true"></span>
       <span class="sr">
         ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
         ${_('Close')}
       </span>
     </button>
+    <div id="help_wrapper">
 
-    <header>
-      <h2>
-	${Text(_('{platform_name} Help')).format(
-	    platform_name=HTML(u'<span class="edx">{}</span>').format(static.get_platform_name())
-	)}
-      </h2>
-      <hr>
-    </header>
+        <header>
+          <h2>
+        ${Text(_('{platform_name} Help')).format(
```

Comment:
Now that this tab has been renamed `Support`  can we use this opportunity to update the title of this section to be `"{platform_name} Support"` as well